README.rst

```diff
@@ -0,0 +1,40 @@
+===========  ====================================================
+Info         NI SystemLink Configuration Utility
+Author       National Instruments
+===========  ====================================================
+
+.. _about_section:
+
+About
+=====
+The *systemlink-configuration-utility* package contains a command line utility for IT professionals to use
+when managing the configuration of an NI `SystemLink Server <https://ni.com/systemlink>`_ installation.
+
+.. image:: https://badge.fury.io/py/ni/systemlink-configuration-utility.svg
+    :target: https://badge.fury.io/py/ni/systemlink-configuration-utility
```

Comment:
Should we remove this until we've actually published?

```suggestion
```

Reply:
I figured it would just be broken until we publish. I was considering publishing a simple 0.0.1 package just to test the whole workflow anyways.

Reply:
That's fair